spectree/models.py

```diff
@@ -54,10 +58,38 @@ class SecureType(str, Enum):
     OPEN_ID_CONNECT = "openIdConnect"
 
 
-class SecuritySchemesData(BaseModel):
+class SecuritySchemeData(BaseModel):
+    """
+    Security scheme data
+    """
+
     type: SecureType = Field(..., description="Secure scheme type")
     name: str = None
     field_in: str = Field(None, alias="in")
     scheme: str = None
     openIdConnectUrl: str = None
     flows: dict = None
```

Comment:
Except `type`, all the other items are optional. So if the user create `SecuritySchemeData(type="http")`, should that be accepted?

Reply:
Fix